week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -20,56 +18,66 @@ function mood() {
   }
 }
 
+console.log(mood(true));
+
 // 2) For any numerical value greater or equal to 10
 //    Hint: use constant `num` and only change isBigEnough.
 //    Variable isBigEnough needs to evaluate to a boolean
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
-
+  var isBigEnough = new Boolean(true);
+  //isBigEnough = 100 >= 10;
   if (isBigEnough) {
     return "num is greater than or equal to 10";
   } else {
     return "num is not big enough";
   }
 }
 
+console.log(greaterThan10());
+
 // 3) For any numerical value greater or equal to 10
 //    Hint: use the Array method sort()
 //    Remember to Google how to use sort method
 
 function sortArray() {
   const letters = ["a", "n", "c", "e", "z", "f"];
   let sortedLetters;
-
+  sortedLetters = letters.sort();
   return sortedLetters;
 }
 
+console.log(sortArray());
+
 // 4) first5 function should return the first 5 elements of array
 //    Hint: use the Array method splice()
 //    Remember to Google how to use splice()
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
+  let sliced = numbers.splice(0, 5);
 
   return sliced;
 }
+console.log(first5());
 
-// 5) get3rdIndex function needs to take an array `arr` and give 
+// 5) get3rdIndex function needs to take an array `arr` and give
 //    back third element value.
 //    For example with array `[1, 2, 3, 4, 5]` it needs to return `4`
 //
 //    Hint: remember that arrays are zero-index based
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr[3];
```

Comment:
Almost right! You have declared the variable 'index' so you can use that in the next line.